### PR TITLE
v8.08.171007 => Bugfix

### PR DESCRIPTION
**Pull request by willemdh**
no big changes. Fix for #70 and more space for fields in 16 color themes

### CODE REVIEW

**Comment by willemdh** [comment]
This seems better. Please someone verify.

File: FireMotD [hunk 1]
```diff
@@ -365,8 +365,7 @@ ExploreMemory () {
     MemTotalB="$(< /proc/meminfo grep MemTotal | awk '{print $2}')"
     MemTotalGB="$(printf "%0.2f\n" "$(bc -q <<< scale=2\;"$MemTotalB"/1024/1024)")"
     ProcpsVersion="$(free --version | awk '{ print $4 }')"
-    VersionComparison 3.3.9 "$ProcpsVersion"
-    
+    VersionComparison 3.3.9 "$ProcpsVersion"
```
Comment (dimon222) [inline FireMotD]: Not sure why its flagged as diff. Newline?

File: FireMotD [hunk 2]
```diff
@@ -1062,101 +1064,105 @@ $CP1$CP1$CP12$CP1H$C1R$CP1$CP1"
...
     fi
     PhpVersion=$(jq -r ".Php.Version" $ExportFile)
     PhpMaxMemory=$(jq -r ".Php.MaxMemory" $ExportFile)
     if [[ ! -z $PhpVersion ]] && [[ "$PhpVersion" != "null" ]] ; then
-        echo -e "$CP1       ${KS}PHP $ES ${VC}Version: ${VCL}$PhpVersion, ${VC}Memory Limit: ${VCL}$PhpMaxMemory"
+        echo -e "$CP1            ${KS}PHP $ES ${VC}Version: ${VCL}$PhpVersion, ${VC}Memory Limit: ${VCL}$PhpMaxMemory"
+    fi
+    ElasticsearchVersion=$(jq -r ".Elasticsearch.Version" $ExportFile)
+    if [[ ! -z $ElasticsearchVersion ]] && [[ "$ElasticsearchVersion" != "Unknown" ]] && [[ "$ElasticsearchVersion" != "null" ]] ; then
+      echo -e "$CP1  ${KS}Elasticsearch $ES ${VC}Version: ${VCL}$ElasticsearchVersion"
```
Comment (dimon222) [inline FireMotD]: Sounds like new functionality (Elasticsearch integration?). Probably worth to separate to another commit.

File: FireMotD (hunk 1 above)
Comment (willemdh) [reply FireMotD]: Removed blank line there i think

File: FireMotD (hunk 2 above)
Comment (willemdh) [reply FireMotD]: It already was in a previous commit for modern. No time to create separate commit for this. Going to merge.
